types/belter/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for belter 1.0
+// Project: https://github.com/krakenjs/belter#readme
+// Definitions by: sly <https://github.com/SlyBouhafs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Device.js function definitions
```

Comment:
```suggestion
export as namespace belter;

// Device.js function definitions
```
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
https://unpkg.com/belter@1.0.142/dist/belter.js